### DISCUSSION TITLE
Submitting bug reports on CI configuration to homebrew-core

### DISCUSSION THREAD

**Discussion by JasonGross** [Homebrew/discussions#5160] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

The bug report template at https://github.com/Homebrew/homebrew-core/issues/new?assignees=&labels=bug&projects=&template=bug.yml says
> Please verify that you've followed these steps. If you cannot truthfully check these boxes, open a discussion at https://github.com/orgs/Homebrew/discussions instead.

I cannot truthfully check the boxes because they are irrelevant to the issue I'm trying to report, which is reproducible on homebrew-core's own CI (which does not seem to run the relevant commands).

The issue is that changes to the coq formula incorrectly [fail the CI](https://github.com/Homebrew/homebrew-core/actions/runs/8070405889/job/22048679268?pr=158180#step:4:88) with
```
  Error: /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.1.0/bin/bundle clean
  ==> Testing math-comp
  ==> /home/linuxbrew/.linuxbrew/opt/coq/bin/coqc -R /home/linuxbrew/.linuxbrew/Cellar/math-comp/1.19.0/lib/coq/user-contrib/mathcomp mathcomp testing.v
  File "./testing.v", line 1, characters 0-43:
  Error:
  Compiled library mathcomp.ssreflect.ssreflect (in file /home/linuxbrew/.linuxbrew/Cellar/math-comp/1.19.0/lib/coq/user-contrib/mathcomp/ssreflect/ssreflect.vo) makes inconsistent assumptions over library Coq.Init.Ltac
```

Whenever Coq is rebuilt, all of its dependencies must be rebuilt as well.  This error message is an indication that a cached version of math-comp was used instead of attempting a rebuild.  (This is the only issue with the PR I submitted, which has been open for over two months with no response from developers.)  How do I report this bug in CI/formula configuration?

**Comment by Bo98** [comment, marked as answer]
> Whenever Coq is rebuilt, all of its dependencies must be rebuilt as well

That's usually an indication that both Coq and math-comp needs a revision bump, e.g. `revision 1`. We don't automatically rebuild everything as 99% of formulae don't have the same requirement math-comp has, and you would need to edit math-comp anyway for a revision bump for people to get the update to math-comp when they run `brew upgrade`.

**Reply by JasonGross** [reply]
Ah, thank you!
